plugins/maven/maven3-server-impl/src/org/jetbrains/idea/maven/server/Maven3ServerEmbedderImpl.java

```diff
@@ -1236,12 +1236,20 @@ private Artifact resolve(@NotNull final Artifact artifact, @NotNull final List<A
       final org.eclipse.aether.impl.ArtifactResolver artifactResolver = getComponent(org.eclipse.aether.impl.ArtifactResolver.class);
       final MyLoggerFactory loggerFactory = new MyLoggerFactory();
       if (artifactResolver instanceof DefaultArtifactResolver) {
-        ((DefaultArtifactResolver)artifactResolver).setLoggerFactory(loggerFactory);
+        try {
+          ((DefaultArtifactResolver)artifactResolver).setLoggerFactory(loggerFactory);
+        } catch (NoSuchMethodError ignore) {
+          // ignore for maven 3.6.0 as maven-resolver 1.3.0 migrated to slf4j
```

Comment:
Could you please make sure the problem is not suppressed, but instead reported in the IDE? (you can use `loggerFactory` instance to report error back to the IDE process).